projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -810,3 +810,37 @@ dataJobExecutionsCleanupTask:
   maximumExecutionsToStore: "100"
```

Comment:
Is there a reason to keep the old settings?

Reply:
What do you mean?

Reply:
Good catch :)